packages/remix/package.json

```diff
@@ -70,7 +70,9 @@
   "devDependencies": {
     "@remix-run/node": "^1.4.3",
     "@remix-run/react": "^1.4.3",
-    "@types/express": "^4.17.14"
+    "@types/express": "^4.17.14",
+    "fastify": "^4.26.2",
+    "typescript": "^5.4.4"
```

Comment:
Hopefully we can use latest tsc for the package. Otherwise, it's typing the exported wrapper for fastify is [problematic](https://github.com/fastify/fastify/issues/4873#issuecomment-1768382933), since the compiler is not sure that the `handler` is acually a `RouteMethodHandler` which it is.

Reply:
We can loosely type our exported wrapper instead of bumping the TS version. We apply TS version updates to all packages together when we do.

Reply:
Ok, I did not know there was a policy or convention with regard to tsc used. As for loosely typing, that will mean they will need to assert like `as RouteMethodHandler` that somehow seems not that ergonomic, but if it's a common (or at leas acceptable) practice, let's put it this  way. Btw, the only reason the fastify package has been added - is to type the handler, we  are otherwise good, since the internals of the request have already been described in the polimorphic request types definitions.